### ISSUE TITLE
Border line since Home Assistant 2022.11

### ISSUE THREAD

**Issue by Drealine** [UI-Lovelace-Minimalist/UI#1082]
Refer commit https://github.com/UI-Lovelace-Minimalist/UI/commit/b1033fd4e13b1d27943b14f4cc9838059605e4e8 

Bug still appear : 

![image](https://user-images.githubusercontent.com/61891683/200178714-be8faefb-581a-4f7b-b31f-699f690bf534.png)

**Comment by nkkfs** [comment]
In `...\config\custom_components\ui_lovelace_minimalist\lovelace\themefiles\minimalist-desktop\minimalist-desktop.yaml`
try to add
 `ha-card-border-style: none`
under
`  # fix added border-lines in 2022.11
  ha-card-border-width: "0px"`

**Comment by basbruss** [comment]
@drealine see my comment under yours to the above mentioned commit

**Comment by MrAJB** [comment]
Experiencing the same issue as @Drealine, checked on multiple (types of) devices, but the border-lines appear on all devices. Clearing cache does not resolve the issue, theme is correctly applied, re-applying the theme does not resolve the issue.

**Comment by Drealine** [comment]
Yeah @MrAJB, clearing cache. Uninstall browser, same issue.

**Comment by MrAJB** [comment]
@basbruss:
1. Checked, it's there in all theme files.
2. This did the trick for me! After reloading the themes, the issue is resolved. Thanks a lot!

**Comment by Drealine** [comment]
@basbruss,   ha-card-border-width: "0px" ok on files but reload themes not resolved the issue.
Maybe reinstall minimalist integration ?

**Comment by Drealine** [comment]
I've used adaptive-dash for information.

**Comment by Drealine** [comment]
Hi @basbruss 
I'm stupid, since few months, i've used a themes folder define in other location. I don't change it since migrate to Lovelace UI integration instead of manual files. 

This issue is fixed for me.

**Comment by hansfredriksen** [comment]
I had the exact same problem.. Weird border lines around my objects. I cleared cache and tried everything here. I found out the problem was that when I cleared cache Home assistant reverted away from the theme. I was running minimalist, and it all looked fine but after clearing cache borders appeared. Turns out I had to go back into settings by pressing the button with my initials in the lower left side, and choose Minimalist theme again from theme. Suddenly it was all ok again! 

So if you are struggling try this!
